### ISSUE TITLE
Conceptual Issue: Symmetric ties ('friends') or Asymmetric ties ('followers')? 

### ISSUE THREAD

**Issue by quadrismegistus** [ComradOrg/Comrad#22]

**Comment by Simon-L** [comment]
Symmetric ties have that immediate feel of reflecting a closer and more actual relation to someone (something? group, collective?), is that what a socialist network wants though? In my opinion the 1:1 mapping of online relationship to what happens in your real life is a terrible defect of modern social networks. It is enforced on us that our online existence must represent the real life one, bringing over the many aspects of having an "identity".

Asymmetric ties, then? "Followers" sounds passive, we don't like that! We could use another word that isn't hierarchical in essence?

I hope we can have a constructive debate about that.
Thank you for your work, happy to help if I can!

**Comment by hawc2** [comment]
Can there be different names for assymetric and symmetric ties, with different levels of attachment? You follow me, your my 'fan' or whatever sounds less passive than follower - but if I follow you back, then we're officially comrades? If we communicate enough, we become real Komrades?

**Comment by marxzuckerburg** [comment]
This might be a good, flexible solution. It's what [Secure Scuttlebutt](https://scuttlebutt.nz) does and I think it works pretty well. They call it "following", "followers", and "friends", which sounds a bit too hierarchical to me. I really like "fan": it reminds me of "stanning" and seems more active and more internet-compatible.

**Comment by quadrismegistus** [comment]
Trying to formalize your idea @hawc2. One idea would be to have/show for any person something like:

* 5 comrades
* 10 fans

That actually means:

* 15 ppl follow this person
* 10 of whom are not followed back ('fans')
* 5 of whom are followed back ('comrades')
* we don't know how many ppl this person follows (unnecessary information, and avoids 3 way system)

What do you think?

There is also the broader question about the social function of these numbers. Has follower count replaced/quantified some kind of cultural capital or stature? Is recording such numbers consistent with socialism and conducive to an equal, classless society? I'm not sure either way, to be honest.

**Comment by hawc2** [comment]
Yeah this is an interesting way to do it, and provides a different sort of metric for popularity. As you say, I do think we should think twice about whether the platform should enable the kind of open commodification of people's profiles affiliated with popularity rankings like these, but it also doesn't need to be as front and center as on accounts like Twitter while still being a part of the process. Like, why exactly do I need to know how many fans you have? I'm more interested in whether you are my comrade or just someone who follows me from a distance . . .

My only other thought for another possible term to use here would be 'ally', perhaps instead of 'fan', or as an intermediary step between fan and comrade. Like just because we follow each other doesn't mean we've ever spoken. But this is getting into the weeds, maybe making it too complicated

**Comment by mutageneral** [comment]
# Definitions of **Fan**

> _From The American Heritage® Dictionary of the English Language, 5th Edition_
> * An ardent devotee; an enthusiast.
> * Etymologies: Short for **fanatic**.

> _From The Century Dictionary_
> * One who is very enthusiastic on the subject of athletic sports, especially base-ball; one who haunts base-ball grounds and base-ball games; a base-ball ‘fiend.’

### Definitions of **Fanatic**

> _From The American Heritage® Dictionary of the English Language, 5th Edition_
> * A person marked or motivated by an extreme, unreasoning enthusiasm, as for a cause.

> _From The Century Dictionary_
> * Same as **fanatical**.
> * A person affected by zeal or enthusiasm, particularly on religious subjects; one given to wild and extravagant notions of religion.

> _From Wiktionary, Creative Commons Attribution/Share-Alike License_
> * Same as **fanatical**.
> * A person who is zealously enthusiastic for some cause, especially in religion.

> _From WordNet 3.0 by Princeton University_
> * Marked by excessive enthusiasm for and intense devotion to a cause or idea.
> * A person motivated by irrational enthusiasm (as for a cause).

### Definitions of **Fanatical**

> _From The American Heritage® Dictionary of the English Language, 5th Edition._
> * Possessed with or motivated by excessive, irrational zeal.

> _From The Century Dictionary_
> * Wild and extravagant in opinions, particularly in religious opinions; extreme, or maintaining opinions in an extreme way; especially, inordinately zealous, enthusiastic, or bigoted.

> _From Wiktionary, Creative Commons Attribution/Share-Alike License_
> * Having an extreme, irrational zeal or enthusiasm for a specific cause.

> _From the GNU version of the Collaborative International Dictionary of English_
> * Characteristic of, or relating to, **fanaticism**; fanatic.

### Definitions of **Fanaticism**

> _From the GNU version of the Collaborative International Dictionary of English_
> * Excessive enthusiasm, unreasoning zeal, or wild and extravagant notions, on any subject, especially religion, politics or ideology; religious frenzy.

> _From WordNet 3.0 by Princeton University_
> * Excessive intolerance of opposing views.

# Definitions of **Ally**

> _From The American Heritage® Dictionary of the English Language, 5th Edition_
> * One that is allied with another, especially by treaty.
> * One in helpful association with another: synonym: partner.
> * An animal that cooperates with another animal of the same species in an alliance.

> _From The Century Dictionary_
> * One united or associated with another by kinship, treaty, or league; a confederate; more particularly, a sovereign or state connected with another by league offensive and defensive, or a subject or citizen of such sovereign or state.
> * An auxiliary; an associate or friend.

> _From the GNU version of the Collaborative International Dictionary of English_
> * One united to another by treaty or league; usually applied to sovereigns or states; a confederate.
> * Anything associated with another as a helper; an auxiliary.
> * Anything akin to another by structure, etc.
> * To unite, or form a connection between, as between families by marriage, or between princes and states by treaty, league, or confederacy; often followed by to or with.
> * To connect or form a relation between by similitude, resemblance, friendship, or love.

> _From WordNet 3.0 by Princeton University_
> * A friendly nation.
> * An associate who provides cooperation or assistance.

###### Source: https://www.wordnik.com/
***

After reading all this, I think "Ally" would be a friendlier and better fitted term for our needs, as @hawc2 suggested.
Ally is also an internet slang term that roughly mean "An heterosexual person who supports LGBT rights" which is "active and internet-compatible" as @marxzuckerburg said it.
